webui-src/app/main.js

```diff
@@ -87,6 +87,9 @@ const Layout = () => {
         }),
         m('.tab-content', vnode.children),
       ]),
+      m('#notification-container'),
+      m('#modal-container'),
```

Comment:
Do we need to move the modal element to inside the main mount? It might interfere with it's vdom since it's using a different render tree.

Reply:
Yeah that would be better I think. Normally we should try to avoid mixing m.mount & m.render in the same tree

Reply:
Done 👍🏼